apps/src/templates/ChallengeDialog.jsx

```diff
@@ -0,0 +1,139 @@
+import BaseDialog from './BaseDialog';
+import LegacyButton from './LegacyButton';
+import PuzzleRatingButtons from './PuzzleRatingButtons';
+import Radium from 'radium';
+import React from 'react';
+
+const stylesGenerator = assetUrl => ({
+  dialog: {
+    top: '20%',
+    border: '5px solid #75699b',
+    borderRadius: 10,
+  },
+  banner: {
+    backgroundPosition: 'top center',
+    backgroundRepeat: 'no-repeat',
+    backgroundImage: `url(${assetUrl('media/dialog/challenge_target.svg')})`,
+    position: 'relative',
+    marginTop: -85,
+    height: 135,
+  },
+  bannerComplete: {
+    backgroundImage: `url(${assetUrl('media/dialog/challenge_target_complete.svg')})`,
+    backgroundPositionY: -16,
+  },
+  content: {
+    color: '#75699b',
+    position: 'relative',
+    textAlign: 'center',
+    marginBottom: 30,
+  },
+  text: {
+    margin: '0px 40px 20px',
+    textAlign: 'center',
+    fontSize: 18,
+  },
+  title: {
+    textAlign: 'center',
+    position: 'absolute',
+    bottom: 0,
+    color: '#fff',
+    backgroundColor: '#75699b',
+    border: '3px solid white',
+    minWidth: '50%',
+    left: '25%',
+    fontSize: '150%',
+    height: 30,
+    lineHeight: '30px',
+  },
+  primaryButton: {
+    float: 'right',
+  },
+  footer: {
+    marginTop: 20,
+    paddingTop: 20,
+    borderTop: '2px solid #ccc',
+  },
+});
+
+const ChallengeDialog = Radium(React.createClass({
+  propTypes: {
+    assetUrl: React.PropTypes.func,
```

Comment:
You should be able to use `import assetUrl from '@cdo/apps/code-studio/assetUrl';` now!

https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/assetUrl.js